src/App.tsx

```diff
@@ -1,12 +1,73 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { GoodsList } from './components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  const showAll = () => {
+    getAll()
+      .then(gottenGoods => {
+        setGoods(gottenGoods);
+      });
+  };
```

Comment:
Yura said we can use then, but mentors want await)

Reply:
yeap, async await gives you a simpler way to work with asynchronous promise-based code